integration_client.py

```diff
@@ -143,10 +143,7 @@ def __init__(self, collection, *args, **kwargs):
 
         replacement_policy = kwargs.pop('replacement_policy', None)
         if not replacement_policy:
-            mirror = MirrorUploader.sitewide(_db)
-            replacement_policy = ReplacementPolicy(
-                mirror=mirror, links=True
-            )
```

Comment:
This would have caused test failures when core was updated, but this bit of code wasn't tested. I found it by searching all the code for `ReplacementPolicy`.